Documentation/operations/upgrade.rst

```diff
@@ -324,7 +324,8 @@ Removed Options
   and is now removed.
 * ``hubble-flow-buffer-size``: This option was deprecated in 1.10 in favor
   of ``hubble-event-buffer-capacity``. It is now removed.
-
+* The deprecated ``hubble-ca-cert`` configmap has been removed. Use
+  ``hubble-ca-secret`` secret instead.
```

Comment:
Please re-add a blank line after this item and before the title of the next subsection.

Reply:
Blank line is added